CWE/events/Italy ahistorical.txt

```diff
@@ -0,0 +1,5248 @@
+  	desc = EVT_8020462_DESC
+  	picture = "carlo_alberto_dalla_chiesa"
+  
+  	fire_only_once = yes
+
+	trigger = {
+		tag = ITA
+		year = 1998
+		month = 7
+		NOT = { has_global_flag = 1992_start_date_flag }
+		
+		government = nationalist_dictatorship 
+		is_vassal = no
+		war = no
+		
+		ruling_party_ideology = nationalist
+		
+		#Sicily
+		owns = 762
+		owns = 763
+		owns = 764
+		
+		has_country_flag = ITA_dalla_Chiesa
+		has_global_flag = ITA_Operation_Tora_Tora
+	}
+
+	mean_time_to_happen = { days = 8 }
+	
+	
+	option = {
+		name = "Greenlight the operation"
+		prestige = 150
+		
+		#Palermo militarization
+		any_pop = { 
+		    limit = { 
+			    type = soldiers 
+				has_pop_religion = secularism
+				location = { 
+				    OR = { 
+					    region = PAP_752 
+						region = TUS_744 
+						region = MOD_739 
+						region = SAR_720 
+					} 
+				}
+			} 
+		    move_pop = 762 
+		} 
+		SIC_763 = {
+		    any_owned = { 
+			    aristocrats = { money = -250 } 
+		        capitalists = { money = -250 } 
+			    bureaucrats = { money = 100 } 
+		        officers = { money = 175 } 
+			    soldiers = { money = 150 } 
+		    }
+		    any_pop = {  
+		        limit = { OR = { type = aristocrats type = capitalists } }
+			    consciousness = 9
+			    militancy = 9
+		    }
+			any_pop = {  
+		        limit = { OR = { strata = middle strata = poor } }
+				ideology = { factor = 0.1 value = nationalist }
+			    consciousness = -6
+			    militancy = -6
+		    }
+			any_pop = {
+			    limit = { type = soldiers } 
+			    ideology = { factor = 0.2 value = nationalist }
+			    reduce_pop = 1.25
+		    }
+		}
+		SIC_760 = {
+		    any_owned = { 
+			    aristocrats = { money = -500 } 
+		        capitalists = { money = -500 } 
+			    bureaucrats = { money = 50 } 
+		        officers = { money = 150 } 
+			    soldiers = { money = 100 } 
+		    }
+		    any_pop = {  
+		        limit = { OR = { type = aristocrats type = capitalists } }
+			    consciousness = 9
+			    militancy = 9
+		    }
+			any_pop = {  
+		        limit = { OR = { strata = middle strata = poor } }
+				ideology = { factor = 0.1 value = nationalist }
+			    consciousness = -6
+			    militancy = -6
+		    }
+			any_pop = {
+			    limit = { type = soldiers } 
+			    ideology = { factor = 0.2 value = nationalist }
+			    reduce_pop = 1.25
+		    }
+		}
+		
+		ai_chance = { factor = 0.9 }
+	}
+
+	option = {
+		name = "Redlight the operation"
+		prestige = -50
+		
+		SIC_763 = {
+			any_pop = {  
+		        limit = { OR = { strata = middle strata = poor } }
+				ideology = { factor = 0.1 value = communist }
+			    consciousness = 6
+			    militancy = 6
+		    }
+		}
+		SIC_760 = {
+			any_pop = {  
+		        limit = { OR = { strata = middle strata = poor } }
+				ideology = { factor = 0.1 value = communist }
+			    consciousness = 6
+			    militancy = 6
+		    }
+		}
+		
+		759 = { add_province_modifier = { name = generic_uprising duration = 3650 } } #Reggio Calabria
```

Comment:
Some of these provinces are not in the trigger check so I think this should be better:

any_owned = { limit = { OR = { province_id = X province_id = Y } } add_province_modifier = { name = generic_uprising duration = 3650 } }